### PR TITLE
 Input and Output Operators in C (Issue Number : #923) 

### PR DESCRIPTION
**Pull request by Kanishk0412**
<hr>

## Description 📜

In the Documentation form, I have demonstrated how to receive user input and produce output using the C programming language. (Issue Number : #923)

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by vasu-1** [comment]
@geetika005  and @Kanishk0412 are you same guy ??

- if no then why you(@geetika005 ) have assigned it yourself because the code has been edited by @Kanishk0412 !!!!
  
- if you are same than i can see some corrections like..
    - for input you can add more functions like gets() , getchar() , 
    - and for output you can add putchar() , puts()
    - you can add more informations about their properties .
   
I hope it will make sense ! 
Thanks

**Comment by Kanishk0412** [comment]
We are not the same I have made this issue first assign it to me you can check it out.

I'll make the changes thanks for reviewing.

**Comment by vasu-1** [comment]
Change the code as suggested and again reopen pr .. You will be assigned to this issue :+1:

**Comment by Kanishk0412** [comment]
Updated Necessary Changes

File: C_CPP/Input & Output Operations/input-&-ouput-operation.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+void main()  
+{  
+
+    puts("I'm GWOC Contributor.");
+    puts("This is my Contribution.");
+
+    return 0;
+}  
+```
+Output:
```
Comment (vasu-1) [inline C_CPP/Input & Output Operations/input-&-ouput-operation.md:73]:
```suggestion
**Output**
```

File: C_CPP/Input & Output Operations/input-&-ouput-operation.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+#include<stdio.h> 
+
+int main()  
+{
+
+    printf("Hello, World!");
+
+    return 0;   
+}
```
Comment (vasu-1) [inline C_CPP/Input & Output Operations/input-&-ouput-operation.md:24]:
```suggestion
int main()  
{
    printf("Hello, World!");
    return 0;   
}
```

- remove this type of unnecessary white spaces from this file.

**Review by vasu-1 — CHANGES_REQUESTED** [review]
### why you have deleted this readme.md ?? 
### don't delete this readme.

- kindly create readme.md file again and also add this line into this readme
     ` * [Input & Output Operation in C](./input-&-ouput-operation.md) `

**Comment by Kanishk0412** [comment]
I have made changes as you have suggested.
Please review it and merge it.

Thanks and Regards,
Kanishk Tyagi

**Comment by Kanishk0412** [comment]
@vasu-1

**Comment by vasu-1** [comment]
Great @Kanishk0412 !
Good to go for merge !!!!
@Suvraneel

**Comment by Saviour1001** [comment]
Great work @Kanishk0412

**Comment by vasu-1** [comment]
@Kanishk0412 close the Respective issue #923

**Comment by Kanishk0412** [comment]
Okay I'll close it. Thanks for merging my branch.

Regards,
Kanishk Tyagi

**Pull request merged**